### PR TITLE
Strapi 3.x compatible version

### PR DESCRIPTION
**Pull request by Saunved**
I have merged Robert's PR and updated the code according to the comments on this PR: https://github.com/yutikom/strapi-provider-email-nodemailer/pull/10

Please ignore the previous PR, I forgot to update the repo link in one place there. I have closed it now.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,30 +12,30 @@
     "nodemailer": "6.3.0",
     "nodemailer-ntlm-auth": "1.0.1"
```
Comment (veitbjarsch) [inline package.json:13]: This dependency can also be removed now.

File: lib/index.js [hunk 2]
```diff
@@ -11,98 +11,25 @@ const _ = require('lodash')
...
-          user: config.username,
-          pass: config.password
-        }
-      })
+  init: (providerOptions = {}, settings = {}) => {
+    if (providerOptions.auth && providerOptions.auth.method === 'NTLM'){
+      providerOptions.customAuth = {
+        NTLM: nodemailerNTLMAuth
+      }
     }
```
[the remark this answers is not in the document]
Comment (roschaefer) [reply lib/index.js:23]: I don't have time to fix this myself in the next days. I :heart: the idea that we remove this dependency and let the user add it when it's really needed.

**Comment by veitbjarsch** [comment]
To Speed this up i've created my own fork and updated these changes. I'll close this pr and merge https://github.com/yutikom/strapi-provider-email-nodemailer/pull/14.

**Comment by Saunved** [comment]
Just saw these changes. Thanks for reviewing the PR and fixing the issues. Appreciate the resolution! :smile:

**Pull request closed without merging**